src/app/applications/datatarget/opendatadk/opendatadk-edit/opendatadk-edit.component.html

```diff
@@ -295,60 +400,60 @@ <h6 class="mt-3">{{ "QUESTION.DATATARGET.RELATIONS" | translate }}</h6>
+                      <mat-select
+                        [(value)]="element.iotDeviceIds"
+                        [compareWith]="payloadDevicesDropdownCompare"
+                        multiple
+                        name="devices"
+                      >
+                        <mat-option class="filter-option" disabled="disabled">
+                          <button (click)="selectAllDevices(i)" class="mat-primary fill text-sm" mat-raised-button>
+                            {{ "QUESTION.DATATARGET.SELECTALLDEVICES" | translate }}
+                          </button>
+                          <button (click)="deSelectAllDevices(i)" class="mat-primary fill text-sm" mat-raised-button>
+                            {{ "QUESTION.DATATARGET.DESELECTALLDEVICES" | translate }}
+                          </button>
+                        </mat-option>
+                        @for (device of devices; track device) {
+                          <mat-option [value]="device.id">{{ device.name }}</mat-option>
+                        }
+                      </mat-select>
+                    </mat-form-field>
+                  </div>
+                </td>
+                <td>
+                  <div class="row">
+                    <mat-form-field appearance="fill">
+                      <mat-label>{{ "QUESTION.DATATARGET.SELECT-PAYLOADDECODER" | translate }}</mat-label>
+                      <mat-select [(value)]="element.payloadDecoderId" matNativeControl name="payloadDecoderId">
+                        <mat-option [value]="0">
+                          {{ "QUESTION.DATATARGET.NO-PAYLOAD-DECODER-SELECTED" | translate }}
                         </mat-option>
-                      }
-                    </mat-select>
-                  </mat-form-field>
-                </div>
-              </td>
-              <td>
-                <a (click)="openDeleteDialog(i)">
-                  <div class="text-center m-2">
-                    <fa-icon [icon]="faTimesCircle"></fa-icon>
-                    <p>{{ "DATATARGET.DELETE" | translate }}</p>
+                        @for (payloadDecoder of payloadDecoders; track payloadDecoder) {
+                          <mat-option [value]="payloadDecoder.id">
+                            {{ payloadDecoder.name }}
+                          </mat-option>
+                        }
+                      </mat-select>
+                    </mat-form-field>
                   </div>
-                </a>
-              </td>
-            </tr>
-          }
+                </td>
+                <td>
+                  <a (click)="openDeleteDialog(i)">
+                    <div class="text-center m-2">
+                      <fa-icon [icon]="faTimesCircle"></fa-icon>
+                      <p>{{ "DATATARGET.DELETE" | translate }}</p>
+                    </div>
+                  </a>
+                </td>
+              </tr>
+            }
           </tbody>
         </table>
       }
```

Comment:
Der er en bug hvor [Creative Commons CC0 1.0 Universel (CC0 1.0) licens](https://creativecommons.org/publicdomain/zero/1.0/deed.da) skal åbne i ny fane i stedet for samme fane